### DISCUSSION TITLE
Suggestion: (Extended) User management

### DISCUSSION THREAD

**Discussion by lejyfps** [diced/zipline#226] [Ideas]
### Suggestion

- Admin option to view ALL *(via the "Administration Tab")* files that have been uploaded
- Possibility to limit users with storage
- Possibility to limit users with file types
- Possibility to "lock" user accounts instead of having to delete them

Would be siiiiick

**Comment by TacticalTechJay** [comment]
- That's something I've pondered over as well.
- I doubt that'll be included, I did attempt it in a fork. (#201)j But dealing with merge conflicts while that issue was open was tough to keep up with.
- That's... a new one, what about admins?
- "lock" like prevent uploading (or even url shortcuts) or logging in?

**Comment by lejyfps** [comment]
> * That's... a new one, what about admins?
> * "lock" like prevent uploading (or even url shortcuts) or logging in?

1. Admins shoult not can lock each other, the only one who should be able to lock such thinggs schould be the "super admin"
2. "lock" like prevent everything even the login (Login msg: "you are banned" or smth) :P

**Comment by diced** [comment]
Not sure of this will be in 3.6.1 but you may see it in 3.7 or 3.6.2.

**Comment by lejyfps** [comment]
> Not sure of this will be in 3.6.1 but you may see it in 3.7 or 3.6.2.

awsome! :)

**Comment by spiritLHLS** [comment]
I see that non-admin users can also set up domains in the dashboard, is there a plan to restrict only admins to do this? Or can administrators set certain permissions for other users to operate the dashboard?
(API included:I want a TOKEN that can only upload files, but there doesn't seem to be a way to do that.)
